problems/2.sum.js

```diff
@@ -28,7 +32,9 @@ function sumThree() {
  * @param {number} b - the second number which is subtracted from the first
  * @returns {number} - Equal to a minus b
  */
-function subtract() {
+function subtract(numA,numB) {
+  Minus = numA - numB
```

Comment:
variables should be camelCase so start with a lowercase and needs to be declared.